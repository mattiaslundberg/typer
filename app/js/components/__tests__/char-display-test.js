import React from 'react'
import CharDisplay from '../char-display'
import renderer from 'react-test-renderer'

describe('CharDisplayTest', () => {
  it('Renders with correct class', () => {
    const component = renderer.create(
      <CharDisplay cssClass="myclass" char="B" />
    );

    expect(component.toJSON()).toMatchSnapshot();
  })
})
