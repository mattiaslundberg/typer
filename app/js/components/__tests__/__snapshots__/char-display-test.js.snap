exports[`CharDisplayTest Renders with correct class 1`] = `
<span
  className="myclass">
  B
</span>
`;
